CRM/Core/BAO/Email.php

```diff
@@ -334,4 +334,29 @@ public static function del($id) {
     return CRM_Contact_BAO_Contact::deleteObjectWithPrimary('Email', $id);
   }
 
+  /**
```

Comment:
gawd, I never knew this was not a boolean!

We do have a precedent for handling custom option lists eg. 
https://github.com/fuzionnz/civicrm-core/blob/f3ea5182292f978844495f7c8662997a57484c73/xml/schema/Campaign/CampaignGroup.xml#L51-L51

```
  <field>
    <name>group_type</name>
    <title>Campaign Group Type</title>
    <type>varchar</type>
    <length>8</length>
    <default>NULL</default>
    <comment>Type of Group.</comment>
    <pseudoconstant>
      <callback>CRM_Core_SelectValues::getCampaignGroupTypes</callback>
    </pseudoconstant>
    <html>
      <type>Select</type>
    </html>
  </field>
```